Makefile

```diff
@@ -107,9 +107,7 @@ update-schema:
 
 .PHONY: update-api
 update-api:
-ifeq "$(LXD_OFFLINE)" ""
-	(cd / ; go install -v -x github.com/go-swagger/go-swagger/cmd/swagger@latest)
-endif
+	command -v swagger >/dev/null || (cd / ; go install -v -x github.com/go-swagger/go-swagger/cmd/swagger@latest)
```

Comment:
Why did you remove the offline check?

Reply:
if its missing and in offline mode wont this now fail?

Reply:
Oh that's right, I didn't properly understood the `ifeq` but in that case, if it's missing and in offline mode, wouldn't the next command fail anyway?

Reply:
The way I understand it, is that if `LXD_OFFLINE` is set, we should only do offline stuff. That includes _not_ running `go install`.

This commit ignores the offline mode as it would pull swagger if it's not available. I don't believe we should do this. Another issue is that it would never update swagger if it's already installed.

Reply:
also if swagger is installed but is out of date it would get updated but with this change it would not. i think we should drop this commit.

Reply:
> The way I understand it, is that if `LXD_OFFLINE` is set, we should only do offline stuff. That includes _not_ running `go install`.
> 
> This commit ignores the offline mode as it would pull swagger if it's not available. I don't believe we should do this. 

Irrespective of the `LXD_OFFLINE` mode, if swagger is not available, the `swagger generate` would have failed, no?

> Another issue is that it would never update swagger if it's already installed.

I wasn't aware that `go install` would update it, thanks. I've dropped the commit.

Reply:
Yes it would have failed